frontend/public/components/dashboard/project-dashboard/project-dashboard.tsx

```diff
@@ -84,7 +83,7 @@ export const ProjectDashboard: React.FC<ProjectDashboardProps> = ({ obj }) => {
             <DevGettingStartedSection userSettingKey="devconsole.projectOverview.gettingStarted" />
           ) : (
             consoleCapabilityGettingStartedBannerIsEnabled && (
-              <GettingStartedSection userSettingKey={PROJECT_OVERVIEW_USER_SETTINGS_KEY} />
+              <DevGettingStartedSection userSettingKey={PROJECT_OVERVIEW_USER_SETTINGS_KEY} />
```

Comment:
There's no need for a perspective check here. Can add a perspective check for userSettingKey. And `consoleCapabilityGettingStartedBannerIsEnabled` is not required for `DevGettingStartedSection`

`<DevGettingStartedSection userSettingKey={perspective === 'dev' ? 'devconsole.projectOverview.gettingStarted' : PROJECT_OVERVIEW_USER_SETTINGS_KEY} />`